packages/rich-text/src/component/index.js

```diff
@@ -40,60 +39,6 @@ export function useRichText( {
 	const ref = useRef();
 	const [ activeFormats = [], setActiveFormats ] = useState();
 
-	/**
-	 * Converts the outside data structure to our internal representation.
-	 *
-	 * @param {*} string The outside value, data type depends on props.
-	 *
-	 * @return {Object} An internal rich-text value.
-	 */
-	function formatToValue( string ) {
-		if ( disableFormats ) {
-			return {
-				text: string,
-				formats: Array( string.length ),
-				replacements: Array( string.length ),
-			};
-		}
-
-		if ( format !== 'string' ) {
-			return string;
-		}
-
-		const result = create( {
-			html: string,
-			multilineTag,
-			multilineWrapperTags:
-				multilineTag === 'li' ? [ 'ul', 'ol' ] : undefined,
-			preserveWhiteSpace,
-		} );
-
-		result.formats = __unstableAfterParse( result );
-
-		return result;
-	}
-
-	/**
-	 * Converts the internal value to the external data format.
-	 *
-	 * @param {Object} val The internal rich-text value.
-	 *
-	 * @return {*} The external data format, data type depends on props.
-	 */
-	function valueToFormat( val ) {
-		if ( disableFormats ) {
-			return val.text;
-		}
-
-		if ( format !== 'string' ) {
-			return;
```

Comment:
`val` should have been returned here, but nothing is returned, so the format prop was also broken.